libraries/APM_Control/AP_PitchController.cpp

```diff
@@ -199,10 +208,13 @@ float AP_PitchController::_get_rate_out(float desired_rate, float scaler, bool d
         rate_pid.set_integrator(old_I);
     }
 
+    // calculate feed forward for auto flaps
+    const float scaled_ff_flap = (SRV_Channels::get_output_norm(SRV_Channel::k_flap_auto) + 1) * _flap_ff * scaler * 0.5f;
```

Comment:
Might need to use  `get_slew_limited_output_scaled` (with the right maths for scale.)

Reply:
I'd actually added: `SRV_Channel::get_slew_limited_output_norm()` in this PR for this exact purpose, but apparently pushed the one not using it (I was switching back and forth in testing).

I wanted to avoid having to do the extra scaling here, because it makes it harder to handle changing the scaling in the future.